.changeset/odd-cats-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+'slate-react': patch
```

Comment:
Probably needs to be minor since it is adding to the API.

Reply:
Oh yeah, you are right